confidant/webhook.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+import logging
+
+from confidant.app import app
+
+
+def send_event(event_type, services, credentials):
+    try:
+        webhook_url = app.config.get('WEBHOOK_URL')
+        if not webhook_url:
+            logging.warning('Failed to find a WEBHOOK_URL in config')
```

Comment:
On second thought, it's likely that most folks won't have webhooks setup. Can you make this a debug, rather than a warning?